packages/tsconfig-reference/copy/id/options/skipLibCheck.md

```diff
@@ -0,0 +1,10 @@
+---
+display: "Melewati pengecekan pustaka"
+oneline: "Melewati pengecekan jenis berkas deklarasi"
+---
+
+Melewati pemeriksaan jenis berkas deklarasi.
+
+Ini dapat menghemat waktu selama kompilasi dengan mengorbankan akurasi dari sistem type. Misalnya, dua pustaka bisa mendefinisikan dua salinan dari `type` (tipe) yang sama dengan cara yang tidak konsisten daripada melakukan pemeriksaan penuh pada semua berkas `d.ts`, Typescript akan mengecek kode tipe secara spesifik pada kode sumber aplikasi anda.
```

Comment:
`mengecek kode tipe secara spesifik` ini konteks bahasa inggrisnya gimana kah?

Reply:
`TypeScript will type check the code you specifically ...` , saya agak bingung antara type untuk type atau ketik.

Reply:
Coba post kalimat utuhnya dalam bahasa Inggris nya

Reply:
Rather than doing a full check of all `d.ts` files, TypeScript will type check the code you specifically refer to in your app's source code.

Reply:
Ini aja nih mas:

> Dibandingkan melakukan pengecekan menyeluruh pada semua berkas `d.ts`, TypeScript akan mengecek tipe data dari kode yang Anda acu secara spesifik di dalam kode sumber aplikasi.

Reply:
ok siap mas, terima kasih koreksinya.